cardano-db-sync/cardano-db-sync.cabal

```diff
@@ -31,17 +31,20 @@ library
                         -Wincomplete-uni-patterns
 
   exposed-modules:      Cardano.DbSync
+                        Cardano.DbSync.Types
+                        Cardano.DbSync.Error
+                        Cardano.DbSync.Util
+                        Cardano.DbSync.Era.Byron.Util
+                        Cardano.DbSync.Era.Shelley.Util
+
```

Comment:
What specifically do you need from those modules. There are a huge amount of internals exposed from those. If you insist on having those exposed you get to keep all the pieces when I break things. Or we could figure out what you actually need and just expose that.

Reply:
I need quite a lot?
Currently, I would leave this exposed and minimize this over time.

Reply:
As long as you are aware that this project is under **HEAVY** development at the moment and anything could change at any time.

I still have not been made aware of what SMASH is or how it interoperates with `db-sync`, nor has my advice been sought on how the interoperate with `db-sync` as smooth as possible.